### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by joshualyngdoh**
# Description
- Page is now Interactive
- Updated README
- Check for linter errors

### CODE REVIEW

File: src/Status.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+    LocalStorage.saveData(tasksList);
+  }
+
+  static clearAllCompletedTask(e, tasksList) {
+    tasksList = LocalStorage.getData();
+    tasksList = tasksList.filter((task) => task.completed === false);
+    LocalStorage.saveData(tasksList);
+  }
+}
```
Comment (youmari) [inline src/Status.js:24]: - [x] when make the tasks status: true and reload the page twice they become `status: false` again please make sure to fix that you can see the bug in the following  video : 
https://www.loom.com/share/f545a61f729145e398400d6314ce58c6

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @joshualyngdoh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/Status.js (hunk 1 above)
Comment (joshualyngdoh) [reply src/Status.js:24]: Thank you for your feedback

**Review by mikethreels — APPROVED** [review]
Hi Josh, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joshualyngdoh** [comment]
Thank you

**Pull request merged**